Library/Formula/git-when-merged.rb

```diff
@@ -0,0 +1,13 @@
+class GitWhenMerged < Formula
+  desc "Find the where a commit was merged in git."
```

Comment:
Why don't you use the description in the repo?

`Determine when a particular commit was merged into a git branch`